Chart/Chart.swift

```diff
@@ -291,6 +297,12 @@ class Chart: UIControl {
                 
                 if series.line {
                     drawLine(xValues: scaledXValues, yValues: scaledYValues, seriesIndex: index)
+                    if series.points {
+                        for i in 1..<scaledXValues.count {
+                            drawPoints(xValues: scaledXValues, yValues: scaledYValues, seriesIndex: index, i: i)
```

Comment:
If I understand, `drawPoints` actually draws only one point for the coordinates at the `i` position. I'm confused by the plural in the method name. Why pass all the `xValues` and `yValues` as arguments?